activesupport/lib/active_support/cache.rb

```diff
@@ -575,12 +575,12 @@ def handle_expired_entry(entry, key, options)
         end
 
         def get_entry_value(entry, name, options)
-          instrument(:fetch_hit, name, options) { |payload| }
+          instrument(:fetch_hit, name, options) { |_| }
```

Comment:
@jeremy fixed. Previously, I had just dropped this block since it does nothing. But its required, forgot to rm this.